src/main/java/com/google/idea/perf/tracer/ui/TracerConfigModel.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.idea.perf.tracer.ui
+
+import com.google.idea.perf.tracer.TraceOption
+import com.google.idea.perf.tracer.TraceTarget
+import javax.swing.table.AbstractTableModel
+import kotlin.math.min
+
+/** The table model for [TracerTable]. */
+class TracerConfigModel : AbstractTableModel() {
```

Comment:
Since we are displaying mostly-static information, would it be simpler to just display a text field that lists the tracing commands issued so far?

Reply:
done